src/libsync/abstractnetworkjob.cpp

```diff
@@ -75,12 +74,15 @@ AbstractNetworkJob::AbstractNetworkJob(AccountPtr account, const QString &path,
 
 void AbstractNetworkJob::setReply(QNetworkReply *reply)
 {
-    if (reply)
+    if (reply) {
         reply->setProperty("doNotHandleAuth", true);
+    }
 
     QNetworkReply *old = _reply;
     _reply = reply;
-    delete old;
+    if (old) {
+        delete old;
+    }
```

Comment:
A bit fishy, deleting nullptr should be safe and otherwise this check doesn't help.